### ISSUE TITLE
Document how to use with kops

### ISSUE THREAD

**Issue by tamalsaha** [appscode/guard#18]
```
kubeAPIServer:
  authenticationTokenWebhookConfigFile: /etc/kubernetes/guard-config
```

**Comment by tamalsaha** [comment]
https://github.com/kubernetes/kops/issues/3258

**Comment by tamalsaha** [comment]
https://github.com/kubernetes/kops/pull/2090

**Comment by tsupertramp** [comment]
Quick setup: - Will pr docs after christmas:
Check ip to use: kops edit cluster --name <cluster_name> 
if nonMasqueradeCIDR: 100.64.0.0/10 => Then ip to use is 100.64.10.96

After everything is generated: ```kops edit cluster```
```
spec: 
  kubeAPIServer:
    authenticationTokenWebhookConfigFile: /srv/kubernetes/webhook-guard-config
  fileAssets:
  - content: |
       (OUTPUT of: guard get webhook-config ivx -o github --addr=100.64.10.96:9844)
    name: guard-github-auth
    path: /srv/kubernetes/webhook-guard-config
    roles:
    - Master
```

Update cluster config: kops update cluster; then kops update cluster --yes

Rolling updater master nodes, one after another. Test with one master then every third request should "work" - So you can test without breaking your setup. ```kops rolling-update cluster staging-185.k8s.ivx.cloud --instance-group master-eu-west-1a--yes``` - If something does not work: Ssh to master and check kube-api logs /var/log/ - They will help you debugging.

**Comment by tamalsaha** [comment]
Closed via #43
